src/Workleap.AspNetCore.Authentication.ClientCredentialsGrant/AuthenticationBuilderExtensions.cs

```diff
@@ -43,6 +48,19 @@ public static AuthenticationBuilder AddClientCredentials(this AuthenticationBuil
             builder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<IPostConfigureOptions<JwtBearerOptions>>(new ClientCredentialsPostConfigureOptions(authScheme)));
         }
 
+        builder.Services.PostConfigure<SwaggerGenOptions>(options =>
+        {
+            if (options.OperationFilterDescriptors.All(x => x.Type != typeof(SecurityRequirementOperationFilter)))
```

Comment:
Not sure if there are a lot of OperationFilterDescriptors of not, wouldn't it have the potential to be more performant looking for .Any that are of said type? Maybe it's not a big enough collection to really be worth tho

Reply:
This is not a code path that is worth optimizing (also there might not be a difference between Any and All) - it's a one-shot configuration call